utils/src/main/scala/za/co/absa/enceladus/utils/types/Defaults.scala

```diff
@@ -52,6 +52,7 @@ object GlobalDefaults extends Defaults {
       case _: DoubleType    => 0.0d
       case _: LongType      => 0L
       case _: StringType    => ""
+      case _: BinaryType    => Array[Byte]()
```

Comment:
Just my personal taste
```suggestion
      case _: BinaryType    => Array.empty[Byte]
```

Reply:
Sure, why not, changed.